### PR TITLE
removing-tool-language

### PR DESCRIPTION
**Pull request by harleenkohli**
proposed fix for spec issue # 825

### CODE REVIEW

File: src/ReleaseHistory.md [hunk 1]
```diff
@@ -263,5 +263,8 @@
...
 
 ## **v2.0.0-csd.2.beta.2019.02.20** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.2.beta.2019.02.20) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.2.beta.2019.02.20) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.2.beta.2019.02.20)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.2.beta.2019.02.20))
-* API BREAKING: Remove 'invocation'attachments', now replaced by 'run.tool.extensions'. https://github.com/oasis-tcs/sarif-spec/issues/327
+* API BREAKING: Remove 'invocation.attachments', now replaced by 'run.tool.extensions'. https://github.com/oasis-tcs/sarif-spec/issues/327
+
+## **v2.0.0-csd.2.beta.2019.02.20** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.2.beta.2019.02.20) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.2.beta.2019.02.20) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.2.beta.2019.02.20)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.2.beta.2019.02.20))
```
Comment (michaelcfanning) [inline src/ReleaseHistory.md]: >v2.0.0-csd.2.beta.2019.02.20* [](start = 5, length = 29)

Don't recap this header. :)

**Comment by michaelcfanning** [comment]
>                     // SARIF TC31. Nothing to do.

Need to author ApplyChangesFromTC32, which should remove tool.language if it finds it. :) Once you have a tool instance, just call tool.Remove("language") on it. See other cases of property removal in this file. Do not set the property value to null, this won't validate. 

---
Refers to: src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs:54 in a7ddfa4. [](commit_id = a7ddfa4b96418389a8e58f0cabbbfa1aae725414, deletion_comment = False)

File: src/ReleaseHistory.md (hunk 1 above)
Comment (harleenkohli) [reply src/ReleaseHistory.md]: oops! :) thanks!

---
In reply to: [259556888](https://github.com/Microsoft/sarif-sdk/pull/1306#discussion_r259556888) [](ancestors = 259556888)

**Comment by harleenkohli** [comment]
>                     // SARIF TC31. Nothing to do.

thanks!

---
In reply to: [466601909](https://github.com/Microsoft/sarif-sdk/pull/1306#issuecomment-466601909) [](ancestors = 466601909)

---
Refers to: src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs:54 in a7ddfa4. [](commit_id = a7ddfa4b96418389a8e58f0cabbbfa1aae725414, deletion_comment = False)

File: src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs [hunk 2]
```diff
@@ -122,9 +126,32 @@ public static class PrereleaseCompatibilityTransformer
...
+            if (tool["language"] is JToken language)
+            {
+                tool.Remove("language");
+            }
+        }
+
         private static bool ApplyChangesFromTC31(JObject sarifLog)
         {
-            UpdateSarifLogVersion(sarifLog);            ;
+            UpdateSarifLogVersion(sarifLog);
```
Comment (michaelcfanning) [inline src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs]: >UpdateSarifLogVersion(sarifLog); [](start = 12, length = 32)

we can remove this call now that we have it in the TC32 helper. I'll do this in the next change!

**Comment by michaelcfanning** [comment]
LGTM, well done

**Pull request merged**